### PR TITLE
chore(ci): add overflowing add and sub to integer benchmarks

### PR DESCRIPTION
**Pull request by soonum**

### CODE REVIEW

File: tfhe/benches/integer/bench.rs [hunk 1]
```diff
@@ -806,11 +808,21 @@ define_server_key_bench_scalar_default_fn!(
     display_name: add,
     rng_func: default_scalar
 );
+define_server_key_bench_scalar_default_fn!(
+    method_name: unsigned_overflowing_scalar_add_parallelized,
+    display_name: add,
```
Comment (tmontaigu) [inline tfhe/benches/integer/bench.rs:813]: shouldn't the display_name be `overflowing_add` ?

File: tfhe/benches/integer/bench.rs [hunk 2]
```diff
@@ -806,11 +808,21 @@ define_server_key_bench_scalar_default_fn!(
...
+    rng_func: default_scalar
+);
 define_server_key_bench_scalar_default_fn!(
     method_name: scalar_sub_parallelized,
     display_name: sub,
     rng_func: default_scalar
 );
+define_server_key_bench_scalar_default_fn!(
+    method_name: unsigned_overflowing_scalar_sub_parallelized,
+    display_name: sub,
```
Comment (tmontaigu) [inline tfhe/benches/integer/bench.rs:823]: shouldn't the display_name be `overflowing_sub` ?

File: tfhe/benches/integer/signed_bench.rs [hunk 3]
```diff
@@ -361,10 +361,18 @@ define_server_key_bench_binary_signed_clean_inputs_fn!(
     method_name: add_parallelized,
     display_name: add
 );
+define_server_key_bench_binary_signed_clean_inputs_fn!(
+    method_name: signed_overflowing_add_parallelized,
+    display_name: add
```
Comment (tmontaigu) [inline tfhe/benches/integer/signed_bench.rs:366]: shouldn't the display_name be `overflowing_add` ?

File: tfhe/benches/integer/signed_bench.rs [hunk 4]
```diff
@@ -361,10 +361,18 @@ define_server_key_bench_binary_signed_clean_inputs_fn!(
...
+);
 define_server_key_bench_binary_signed_clean_inputs_fn!(
     method_name: sub_parallelized,
     display_name: sub
 );
+define_server_key_bench_binary_signed_clean_inputs_fn!(
+    method_name: signed_overflowing_sub_parallelized,
```
Comment (tmontaigu) [inline tfhe/benches/integer/signed_bench.rs:373]: shouldn't the display_name be `overflowing_sub` ?

File: tfhe/benches/integer/signed_bench.rs [hunk 5]
```diff
@@ -819,11 +841,21 @@ define_server_key_bench_binary_scalar_clean_inputs_fn!(
     display_name: add,
     rng_func: default_scalar
 );
+define_server_key_bench_binary_scalar_clean_inputs_fn!(
+    method_name: signed_overflowing_scalar_add_parallelized,
+    display_name: add,
```
Comment (tmontaigu) [inline tfhe/benches/integer/signed_bench.rs:846]: shouldn't the display_name be `overflowing_add` ?

File: tfhe/benches/integer/signed_bench.rs [hunk 6]
```diff
@@ -819,11 +841,21 @@ define_server_key_bench_binary_scalar_clean_inputs_fn!(
...
+    rng_func: default_scalar
+);
 define_server_key_bench_binary_scalar_clean_inputs_fn!(
     method_name: scalar_sub_parallelized,
     display_name: sub,
     rng_func: default_scalar
 );
+define_server_key_bench_binary_scalar_clean_inputs_fn!(
+    method_name: signed_overflowing_scalar_sub_parallelized,
+    display_name: sub,
```
Comment (tmontaigu) [inline tfhe/benches/integer/signed_bench.rs:856]: shouldn't the display_name be `overflowing_sub` ?

File: tfhe/benches/integer/bench.rs (hunk 1 above)
Comment (soonum) [reply tfhe/benches/integer/bench.rs:813]: It would be more clear you're right.

**Review by IceTDrinker — APPROVED** [review]
Looks good I think !

**Pull request merged**